.github/workflows/publish.yml

```
@@ -56,18 +46,21 @@ jobs:
         run: |
           sudo apt update
           sudo apt install -y libcairo2-dev libjpeg-dev libpango1.0-dev libgif-dev librsvg2-dev
+
       - run: npm install
+
       - run: npm publish --access public
 
   notify:
     name: Notify
     needs: [publish]
-    if: ${{ !cancelled() }}
+    if: !cancelled()
```

```suggestion
    if: ${{ !cancelled() }}
```

https://github.com/pcvg/zvezdochki/actions/runs/19902411975
Unexpected tag '!cancelled()'